packages/perspective/package.json

```diff
@@ -22,9 +23,11 @@
     "build": "npm-run-all build:compile:* build:copy:* build:webpack",
     "build:compile:copy": "mkdir -p obj build",
     "build:compile:emmake": "cd obj/ && emcmake cmake ../ && emmake make -j${PSP_CPU_COUNT-8}",
-    "build:copy:async": "cp obj/psp.async.wasm build/psp.async.wasm",
-    "build:copy:sync": "cp obj/psp.sync.wasm build/psp.sync.wasm",
-    "build:webpack": "npm-run-all -p build:webpack:*",
+    "build:copy:asm": "cp obj/psp.asmjs.js build/psp.asmjs.js",
+    "build:copy:async": "cp obj/psp.async.wasm build/psp.async.wasm && cp obj/psp.async.js build/psp.async.js",
+    "build:copy:sync": "cp obj/psp.sync.wasm build/psp.sync.wasm && cp obj/psp.sync.js build/psp.sync.js",
```

Comment:
These files are large.  If we must have them (see summary review), then why bother writing them to `obj/` at all, rather than modify `CMakeFiles.txt` to directly output to `build` as was done previously?

Reply:
This was an accidental commit from earlier work. I can revert.